docs/creating-multinode-controlplane-clusters.md

```diff
@@ -38,27 +56,72 @@ Now that you have a cluster with a single instance control plane, more control p
+  *Pro tip* 💡: You may refer to the machine's YAML in the release to create the
+  above YAML for yourself.
+
+  Fill in the cluster name and update the namespace if necessary. Then copy
+  `control-plane-machine.yaml` as many times as the number of *new* control
+  planes you want to add to your cluster.
+
+  For each copy, edit the file with a unique machine name and create the machine object:
+
+    ```bash
+    kubectl apply -f control-plane-machine.yaml
+    ```
+
+  Note: You may have to use the `-n <NAMESPACE>` option to kubectl in case your
+  cluster and machines were created in the non-default namespace.
+
+2. Optionally watch machines get created
 
-Just like that, you have now transformed your cluster from a single instance controlplane to one with a multi-node controlplane. 🎉
+  Watch the newly created machine objects being reconciled by following the logs
+  from the `aws-provider-controller-manager-0` pod using this command:
+
+    ```bash
+    kubectl -n aws-provider-system logs -f aws-provider-controller-manager-0
+    ```
+
+   Additionally you can view the new instances initializing in the AWS console.
+
+3. Verify nodes are ready
+
+  Once the machine objects have been reconciled and the instances in AWS have
+  been created and initialized, `kubectl get nodes` will show the new
+  machines with master roles.
+
+Your cluster now has a multi-node control plane! 🎉
 
 ## Troubleshooting
+
 ### Instances for the new machines not getting created
-Inspect the logs from the `aws-provider-controller-manager-0` pod to determine why the controlplane is unable to create the necessary resources. Follow the logs from the `aws-provider-controller-manager-0` pod using the command below:
+
+Inspect the logs from the `aws-provider-controller-manager-0` pod to determine
+why the control plane is unable to create the necessary resources. Follow the
+logs from the `aws-provider-controller-manager-0` pod using the command below:
+
 ```bash
 kubectl -n aws-provider-system logs -f aws-provider-controller-manager-0
 ```
-### Instances were created and initialized, but nodes don't show up on running 'kubectl get no'
-Inspect the system log on the newly created instances for information about why this instance failed to join the existing controlplane. Instructions on how to view the system log are available [here](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instance-console.html)
+
+### Instances were created and initialized, but nodes don't show up on running 'kubectl get nodes'
+
+Inspect the system log on the newly created instances for information about why
+this instance failed to join the existing control plane. Instructions on how to
+view the system log are available
+[here](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instance-console.html).
+
+## High availability vs multi-node control plane
+
+We don't call a multi-node control plane highly available in this document
```

Comment:
Thinking if we should call this out early in the doc.🤔

Reply:
Yeah, makes sense to me.